Facebook.Unity/Results/GraphResult.cs

```diff
@@ -50,19 +50,16 @@ private void Init(string rawResult)
                 return;
             }
 
-            object serailizedResult = MiniJSON.Json.Deserialize(this.RawResult);
-            var jsonObject = serailizedResult as IDictionary<string, object>;
-            if (jsonObject != null)
+            if (this.ResultDictionary == null)
```

Comment:
actually, this PR is no longer needed as of 7.9.0. I guess we have been using an older version of the SDK.